common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -136,8 +154,6 @@ function (HTML5Video, Resizer) {
         // http://code.google.com/p/gdata-issues/issues/detail?id=4654
         // When the YouTube team fixes the API bug, we can remove this
         // temporary bug fix.
```

Comment:
Thanks, done.  I thought about that, but didn't move it, since I felt the comment was in slightly the wrong place even before my change, and I didn't want to clutter up the PR with unrelated clean-ups.  After my change, it is *completely* in the wrong place, so it actually does make sense to include that change.